wechat/handlers.py

```diff
@@ -1,10 +1,20 @@
 # -*- coding: utf-8 -*-
 #
 from wechat.wrapper import WeChatHandler
+from wechat.models import Activity, Ticket
+from WeChatTicket.settings import WECHAT_TOKEN, WECHAT_APPID, WECHAT_SECRET
+from wechat.wrapper import WeChatLib
+import threading
+from datetime import datetime
+import time
+import random
+import string
+
 
 
 __author__ = "Epsirom"
 
+remainTicketsLock = threading.Lock()
```

Comment:
i don't think lock threading is a good idea. it's waste a lot of computing resource. maybe just lock the data.